### ISSUE TITLE
Inline messages have no status messages for actions

### ISSUE THREAD

**Issue by Magsu** [sl-design-system/components#1557]
### Provide a general summary of the issue here

When closing an inline message, the user is not notified in any way when the message is closed. In the dynamic variant, where messages can be added and deleted the user is not notified when a message is added or removed, which might cause the user to think the system is dysfunctional and at the least would require the user to start searching whether anything changed in the page.

### 🤔 Expected Behavior?

When a message is removed or added, a screen reader user should have confirmation, that a message was removed or added.

When closing the element, the user’s context should either change (user is aware of the change through change in context after the button was pressed, e.g. focus moves to next heading) or the user should receive a status message confirming the message was closed.

### 😯 Current Behavior

User receives no status for closing, adding or removing a message.

### 💁 Possible Solution

Status messages for screen readers can be provided by using regions with the role “aria-live”. A proper status message would be “message [message heading] closed/added/removed”.

A common method is to have an invisible paragraph with the role "status" (DON'T give it aria-hidden="true"). The content is then changed based on the actions the user takes to give automatic status messages. 

Currently the message elements have a role of “status” but this is not automatically read, because the elements are created with the role. This role expects a change in the children of the element with this role.

### 🖥️ Steps to Reproduce

1. Open a page with element (e.g. [Inline message Dynamic](Open https://storybook.sanomalearning.design/?path=/story/feedback-status-inline-message--dynamic)
2. Start a screen reader
3. Use the buttons Add Message, Remove Message and Close ("X") on the messages

### What browsers are you seeing the problem on? (only for bugs in code)

Chrome

### What operating system are you using?

Windows 10

### 👤 Your name

Valtteri Järvinen

### 🧢 Your product/team

External

### 🎨 Your Theme(s)

Sanoma Learning
```[tasklist]
### Tasks
- [ ] aria-live is missing
- [ ] update documentation: actions within the inline message are the help the user. Ifan action is crucial, use a dialog instead of an inline message
```

**Comment by Diaan** [comment]
Closed in #1699
